addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -150,21 +150,26 @@ def _compute_plan_id(self):
     def _compute_plan_date(self):
         self.plan_date = False
 
+    def _get_summary_lines(self, templates):
+        self.ensure_one()
+        summaries = []
+        for template in templates:
+            summary_line = template.activity_type_id.name
+            if template.summary:
+                summary_line += f": {template.summary}"
```

Comment:
Also do we really trust this to be "markup safe"? Otherwise I'd keep the previous implementation of inserting it inside a markup object as an unsafe string. The goal being you don't get a summary `<script> alert("pwned") </scipt>` and mark it as "safe markup" to be inserted in the DOM

Reply:
I just move this part. I'm not sure about it but I think if it's merged I hope it's safe.

Reply:
I changed it according to the documentation I hope now it's good :raised_hands: